docs/usage.rst

```diff
@@ -447,7 +451,8 @@ token
 
 To set an authorization token, you can set:
 
-- a key named ``gitea_{host}`` in the keyfile, where ``host`` is all-lowercased host name
+- an entry in your ``netrc`` file for the host
```

Comment:
Use of ``netrc`` file should be ordered last as it's the least specific way of providing the credentials.